### DISCUSSION TITLE
Missing EXIF or Information editing function

### DISCUSSION THREAD

**Discussion by DamageIncorporated** [ente-io/ente#2359] [Q&A] [answered]
Hey fellow duck-hunters ;-)

I remember when starting with ente a few weeks ago, i was able to select one or more photos, klick on an ( i ) for information symbol or menu entry and then edit EXIF data like a description for one or more photos. 

I also started to add #hashtags in the description after reading some hint around here (can't find that anymore).

Now when I select one or more photos, I can't find that feature anymore. Not in the desktop-app, not on the web-app (which are largely the same, but who knows...). 

Do I miss something, is there an issue with my eyesight, or is this a regression?

I really like to add meta info to my photos to be able to search more specifically in the future and would be very sad if that feature was broken or removed :(

Thank you for any advice or help :)

Also thanks to the team for helping me to degoogle myself!

**Comment by mnvr** [comment, marked as answer]
You'll need to open the image. In the image viewer you'll see an info button that should take you to the screen that you remembered 🙂

<img width="425" alt="Screenshot 2024-07-04 at 14 27 28" src="https://github.com/ente-io/ente/assets/24503581/515acbdf-b7cd-4cab-b649-043917b0ef76">

**Reply by DamageIncorporated** [reply]
Ahh I see the light.

Then I just have to file feature request to allow bulk editing of EXIF-Data ;-) 
Which I know is not trivial, but would be very helpful ;-)

**Reply by mnvr** [reply]
While you can't bulk edit arbitrary EXIF data yet, you can bulk edit times (which seems to be the most common use case).

When you select a bunch of files, you should see this "Fix time" option in the selection actions.

<img width="390" alt="Screenshot 2024-07-04 at 14 35 35" src="https://github.com/ente-io/ente/assets/24503581/3778eb9c-f07d-4abc-aee4-d1c6f730d548">
